packages/dds/tree/src/simple-tree/api/simpleSchemaToJsonSchema.ts

```diff
@@ -185,6 +201,13 @@ function convertMapNodeSchema(schema: SimpleMapNodeSchema): JsonMapNodeSchema {
 					},
 		},
 	};
+
+	// Only include "description" property if it is present in the input.
+	if (schema.metadata?.description !== undefined) {
+		output.description = schema.metadata.description;
+	}
```

Comment:
For reference, I was going to suggest if this could be done at the callsite that switches between `convertMapNodeSchema` and its object/array/leaf counterparts... but since this doesn't apply to leaves I ended up finding it a bit weird to do there with an extra condition to exclude leaves. I like the full conversion being encapsulated in the functions 👍.

Reply:
Yeah. Noah made the point earlier that we really need a "copy if defined" helper that would apply this pattern automatically. But I'm calling that out of the scope of this PR.